### ISSUE TITLE
BoundsY example broken

### ISSUE THREAD

**Issue by jordansamuels** [holoviz/holoviews#5628]
TL;DR: The BoundsY example at https://holoviews.org/reference/streams/bokeh/BoundsY.html doesn't work.
See [this pyviz gitter thread](https://matrix.to/#/!daeKOCRnzbteLhAZws:gitter.im/$t79lO4jgbmh78pUROkdLo2k5MjF1fqzlo3VYF34P9zM?via=gitter.im&via=matrix.org).

#### ALL software version info
(this library, plus any other relevant software, e.g. bokeh, python, notebook, OS, browser, etc)
* Holoviews 1.15.4
* Bokeh 2.4.3
* notebook from https://holoviews.org/reference/streams/bokeh/BoundsY.html
#### Description of expected behavior and the observed behavior
* OS MacOS Catalina 10.15.7
* Browser `Version 110.0.5481.100 (Official Build) (x86_64)`

#### Complete, minimal, self-contained example code that reproduces the issue
This comes from https://holoviews.org/reference/streams/bokeh/BoundsY.html .

```
# [ first cell ]
import numpy as np
import holoviews as hv
from holoviews import opts, streams

hv.extension('bokeh')

# [ second cell ]
xs = np.linspace(0, 1, 200)
ys = xs*(1-xs)
curve = hv.Curve((xs,ys))

bounds_stream = streams.BoundsY(source=curve,boundsy=(0,0))

def make_area(boundsy):
    return hv.Area((xs, np.minimum(ys, boundsy[0]), np.minimum(ys, boundsy[1])), vdims=['min','max'])


def make_items(boundsy):
    times = ["{0:.2f}".format(x) for x in sorted(np.roots([-1,1,-boundsy[0]])) + sorted(np.roots([-1,1,-boundsy[1]]))]
    return hv.ItemTable(sorted(zip(['1_entry', '2_exit', '1_exit', '2_entry'], times)))

area_dmap = hv.DynamicMap(make_area, streams=[bounds_stream])
table_dmap = hv.DynamicMap(make_items, streams=[bounds_stream])

(curve * area_dmap + table_dmap).opts(
    opts.Layout(merge_tools=False),
    opts.Overlay(tools=['ybox_select'], active_tools=['ybox_select'])
)

```

#### Stack traceback and/or browser JavaScript console output
```
WARNING:param.OverlayPlot01465: Tool of type 'ybox_select' could not be found and could not be activated by default.
```
#### Screenshots or screencasts of the bug in action
![image](https://user-images.githubusercontent.com/610909/220952070-3a16e5d7-74d7-446e-bf0f-a044069de395.png)
